project/AppDependencies.scala

```diff
@@ -3,20 +3,20 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.20.0",
-    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "5.20.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "3.4.0-play-28",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28" % "0.68.0"
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "7.1.0",
+    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "7.1.0",
+    "uk.gov.hmrc" %% "play-frontend-hmrc" % "3.22.0-play-28",
```

Comment:
Is there any reason for not updating to [3.23.0-play-28](https://github.com/hmrc/play-frontend-hmrc/releases/tag/v3.23.0)?